### PR TITLE
feat: JWT Token on login and register

### PR DESCRIPTION
**Pull request by dlanaras**

### CODE REVIEW

File: backend/.gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
-node_modules/*
\ No newline at end of file
+node_modules/*
+resources/config.json
```
Comment (laurinneff) [inline backend/.gitignore:2]: y no newline

File: backend/resources/config.example.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "JWTSecret": "ADD YOU JWT SECRET (Generate with openssl)"
+}
```
Comment (laurinneff) [inline backend/resources/config.example.json:3]: y no newline

File: backend/tokenGenerator.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+      id: user.id,
+      username: user.username,
+    },
+    secret,
+    {
+      expiresIn: '1d',
+    }
+  );
+  return token;
+}
```
Comment (laurinneff) [inline backend/tokenGenerator.js:17]: y no newline

**Review by laurinneff — DISMISSED** [review]

File: backend/.gitignore (hunk 1 above)
Comment (dlanaras) [reply backend/.gitignore:2]: We aren't using newlines on file end

Comment (laurinneff) [reply backend/.gitignore:2]: yes we are

Comment (ItzSveno) [reply backend/.gitignore:2]: tf bro we do need a newline

**Pull request merged**